pkg/types/helm/providence.go

```diff
@@ -0,0 +1,132 @@
+//
+// Copyright 2021 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helm
+
+import (
+	"bytes"
+	"io"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+
+	"golang.org/x/crypto/openpgp"
+	"golang.org/x/crypto/openpgp/armor"
+	"golang.org/x/crypto/openpgp/clearsign"
```

Comment:
Are there any non `x/crypto` modules we could use. x/crypto is going to get rejected by the FIPS folks. 

Gah, I can't wait to kill pgp with fire (but I totally understand you're working with what you have here).

Reply:
You want to use `github.com/ProtonMail/go-crypto/openpgp` instead.

https://github.com/ProtonMail/go-crypto/

Reply:
I recall @bobcallaway used the protonmail module as well in a different type, @sabre1041 can you look to using protonmail/gro-crypto instead?

Reply:
@lukehinds Looked into `github.com/ProtonMail/go-crypto/openpgp` and it does provide what would be needed for the Helm type. Are you also looking at replacing other instances of `x/crypto` that is in `pki/pgp` too?

Reply:
@sabre1041 , yes very much. I will lodge an issue right now. Also I just noticed I approved when I meant to just comment, please hang out for an approval from mr @bobcallaway as he is the types man.

Reply:
Do we want to separate that work into a separate PR per the issue you will create? I'd be happy to also take a look at that as well

Reply:
the others are in ./pkg/pki/ssh ./pkg/pki/pgp, I will create an issue now, it would be preferable to get this patch to merge in with this already in good order.